eessi/testsuite/hooks.py

```diff
@@ -463,7 +467,7 @@ def filter_valid_systems_by_device_type(test: rfm.RegressionTest, required_devic
     cause the valid_systems to be set to an empty string, and consequently the
     test.valid_systems to an invalid system name (eessi.testsuite.constants.INVALID_SYSTEM).
     """
-    is_cuda_module = is_cuda_required_module(test.module_name)
+    is_cuda_module = is_cuda_required_module(test.module_names)
```

Comment:
I'm getting:

```
WARNING: skipping test 'EESSI_MetalWalls_MW': attribute error: ../../hooks.py:470: 'EESSI_MetalWalls_MW' object has no attribute 'module_names'
    is_cuda_module = is_cuda_required_module(test.module_names)
 (rerun with '-v' for more information)
```

When I run with `reframe --run -t CI -t 1_8_node --system snellius:rome`. Not sure entirely what's wrong here, but seems inconsistent adaptation of that new name?